index.js

```diff
@@ -4,13 +4,36 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "red"
 
 /***** Deliverable 3 *****/
-// console.log("the player object looks like this:", player)
+console.log("the player object looks like this:", player)
+const playerName = document.querySelector("h2")
+playerName.textContent = player.name 
+
+const playerNickname = document.querySelector("em")
+playerNickname.textContent = player.nickname
+
+const img = document.querySelector("img")
+img.src = player.photo
+img.alt = "a Picture of MO"
```

Comment:
It might be helpful to wrap this code up into a function, that will help clean up the code and make it easier if you need to display other players as well - but overall the approach looks good!